src/main/java/net/imagej/launcher/ClassLauncher.java

```diff
@@ -74,6 +74,9 @@
 	 */
 	public static void main(final String[] arguments) {
 		originalArguments = arguments;
+		if (Boolean.getBoolean("imagej.splash") || System.getProperty("imagej.splash") == null) {
+			SplashScreen.show();
```

Comment:
This might be paranoid, but we may want to wrap the `SplashScreen.show()` with a `try { ... } catch (Throwable t) { t.printStackTrace(); }` because we really don't want any bugs in the splash screen to prevent ImageJ from starting up.